bundles/org.openhab.binding.electroluxappliances/src/main/java/org/openhab/binding/electroluxappliances/internal/dto/ApplianceDTO.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.electroluxappliances.internal.dto;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link ApplianceDTO} class defines the DTO for the Electrolux Appliances.
+ *
+ * @author Jan Gustafsson - Initial contribution
+ */
+@NonNullByDefault
+public class ApplianceDTO {
```

Comment:
Nit: applies to all DTO's they can be refactored to a `record`. It would make all the getters and setters obsolete.

Reply:
Noted, will leave the code as it is.